docs/ops/arithmetic/Sqrt_1.md

```diff
@@ -2,36 +2,53 @@
 
 **Versioned name**: *Sqrt-1*
 
-**Category**: Arithmetic unary operation 
+**Category**: Arithmetic unary operation
 
-**Short description**: *Sqrt* performs element-wise square root operation with given tensor.
+**Short description**: Square root element-wise operation.
 
-**Attributes**:
+**Detailed description**: *Sqrt* performs element-wise square root operation on a given input tensor `a`, as in the following mathematical formula, where `o` is the output tensor:
+
+\f[
+o_{i} = \sqrt{a_{i}}
+\f]
 
-    No attributes available.
+
```

Comment:
Let's add an explicit statement that if the input element is negative then the output is undefined.

Reply:
Done